pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1980,3 +2005,33 @@ func ValidateDuplicateDHCPPrivateOptions(PrivateOptions []v1.DHCPPrivateOptions)
 	}
 	return nil
 }
+
+func validateVMINodeNameLabel(oldVMI *v1.VirtualMachineInstance, newVMI *v1.VirtualMachineInstance) *metav1.StatusCause {
```

Comment:
Can we generalize this a little bit? In general, all labels which start with keys of the form `kubevirt.io/*` are not allowed to be modified. Let's ensure we eliminate the whole set of problems associated with this.

Reply:
That's a good idea. I've updated the PR